controllers/ccruntime_controller.go

```diff
@@ -142,6 +158,31 @@ func (r *CcRuntimeReconciler) processCcRuntimeDeleteRequest() (ctrl.Result, erro
 	return ctrl.Result{}, nil
 }
 
+func handlePostUninstall(r *CcRuntimeReconciler) (error, ctrl.Result) {
```

Comment:
Can you swap the return values to be in sync with other functions (ctrl.Result, error)? Makes it easier to read :-)

Reply:
yes, makes sense, I'll change it

Reply:
I changed that now